### ISSUE TITLE
Conjunctival hyperpigmentation

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10880]
Preferred term label:
Conjunctival hyperpigmentation

Synonyms
Hyperpigmentation of the conjunctiva, Pigmented conjunctiva, Conjunctival melanosis

Definition (free text, please give PubMed ID)
Increased pigmentation of the conjunctiva, characterized by brown or black discoloration due to melanocytic proliferation or pigment deposition. It may be associated with benign conditions such as primary acquired melanosis or malignant transformation into conjunctival melanoma.

Parent term (use hpo.jax.org/app)
Abnormal conjunctiva morphology (HP:0000502)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Primary acquired melanosis, Conjunctival melanoma

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001234
- label: Conjunctival hyperpigmentation
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: PMID:34424954
- def: Increased pigmentation of the conjunctiva, characterized by brown or black discoloration due to melanocytic proliferation or pigment deposition. It may be associated with benign conditions such as primary acquired melanosis or malignant transformation into conjunctival melanoma.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10880
- synonyms: Hyperpigmentation of the conjunctiva(NONE)|Pigmented conjunctiva(NONE)|Conjunctival melanosis(NONE)
